nemo/collections/tts/models/glow_tts.py

```diff
@@ -280,3 +280,16 @@ def list_available_models(cls) -> 'List[PretrainedModelInfo]':
         )
         list_of_models.append(model)
         return list_of_models
+
+    def load_state_dict(self, state_dict, strict=True):
+        """Stopgap measure to keep old checkpoints working before full model deprecation.
+
+        This override fiddles with the state_dict in order to keep functionality from old checkpoints after the
+        switch from torch_stft. It will be removed when this model is deprecated.
+        """
+        if 'preprocessor.featurizer.stft.forward_basis' in state_dict:
+            window_dim = state_dict['preprocessor.featurizer.stft.forward_basis'].shape[-1]
+            state_dict['preprocessor.featurizer.window'] = torch.hann_window(window_dim)
```

Comment:
Let's add a warning message that we are defaulting to `hann` window as that might not be the case for all models

Reply:
Good call, added to all four model overrides